Presentation.Web/app/components/data-processing/tabs/data-processing-registration-tab-main.view.html

```diff
@@ -1,72 +1,126 @@
 ﻿<div class="object-section-container">
     <h4 data-element-type="dpaMainDetailHeader">{{vm.headerName}}</h4>
     <div class="panel panel-default">
-        <div class="row">
-            <div class="col-xs-6">
-                <!-- Name -->
-                <div class="form-group">
-                    <label for="dataProcessingRegistration-name" class="control-label small">Navn</label>
-                    <input type="text" id="dataProcessingRegistration-name" class="form-control input-sm" required
-                           data-ng-model="vm.dataProcessingRegistration.name"
-                           data-element-type="dpaMainName"
-                           data-field="name"
-                           ng-blur="vm.changeName(vm.dataProcessingRegistration.name)"
-                           data-ng-disabled="!vm.hasWriteAccess"
-                           autofocus
-                           maxlength="150">
+        <!--1st panel-->
+        <div class="well side-margin-10px top-margin-10px no-padding-bottom">
+            <!-- Name -->
+            <div class="form-group">
+                <label for="dataProcessingRegistration-name" class="control-label small">Navn</label>
+                <input type="text" id="dataProcessingRegistration-name" class="form-control input-sm" required
+                       data-ng-model="vm.dataProcessingRegistration.name"
+                       data-element-type="dpaMainName"
+                       data-field="name"
+                       ng-blur="vm.changeName(vm.dataProcessingRegistration.name)"
+                       data-ng-disabled="!vm.hasWriteAccess"
+                       autofocus
+                       maxlength="150">
```

Comment:
Think you need to put the different sections in their own gray container like on it.-contracts..

right now it's "gray on gray"
![image](https://user-images.githubusercontent.com/13116889/211309387-478c27fe-f04e-4267-bf4c-597181328041.png)